lib/Client/Util.php

```diff
@@ -26,9 +26,27 @@ public function randInt32() {
         return $this->_rng->int32();
     }
 
-    public function nowMicros() {
+    public static function nowMicros() {
         // Note: microtime returns the current time *in seconds* but with
         // microsecond accuracy (not the current time in microseconds!).
         return floor(microtime(TRUE) * 1000.0 * 1000.0);
     }
+
+    public function pushWithMax(&$arr, $item, $max) {
```

Comment:
Can you add a comment to (or rename) this function? It's use isn't immediately clear to me.

Reply:
done